evaluate/metrics/assessment.go

```diff
@@ -99,8 +100,10 @@ func (a Assessments) Score() (score uint) {
 		return 0
 	}
 
-	for _, value := range maps.Values(a) {
-		score += value
+	for key, value := range a {
+		if pointsPerAssessment[key] != 0 {
```

Comment:
Why the change? The value is 0 for `processing-time` so it shouldn't really matter?

Reply:
We don't want to give models a higher score because they take longer to answer.